src/main/scala/de/upb/cs/swt/delphi/instanceregistry/connection/Server.scala

```diff
@@ -579,6 +580,26 @@ object Server extends HttpApp
     }
   }
 
+  /**
+    * Called to run a command in a  docker container. The Id an Command is the required parameter there are other optional parameter can be passed
+    * a query with required parameter Command and Id (so the resulting call is /delete?Id=42&Command=ls).
+    * @return Server route that either maps to 200 Ok or the respective error codes.
+    */
+  def runCommandInContainer() : server.Route = parameters('Id.as[Long], 'Command.as[String],
+    'AttachStdin.as[Boolean].?, 'AttachStdout.as[Boolean].?,
+    'AttachStderr.as[Boolean].?,'DetachKeys.as[String].?, 'Privileged.as[Boolean].?,'Tty.as[Boolean].?, 'User.as[String].?
+    ) { (id, command, attachStdin, attachStdout, attachStderr, detachKeys, privileged, tty, user) =>
+    post {
+        log.debug(s"POST /command has been called")
+        handler.handleCommand(id, command, attachStdin, attachStdout, attachStderr, detachKeys, privileged, tty, user) match {
+          case handler.OperationResult.Ok =>
+            complete{HttpResponse(StatusCodes.OK)}
```

Comment:
Please also handle these cases in the match-case-statement:  
```OperationResult.IdUnknown``` -> Send Response with status code 404 and message "Id not found"
```OperationResult.NoDockerContainer``` -> Send Response with code 400 Bad Request and message "No docker container"